### DISCUSSION TITLE
lutris/proton-ge broke my qtile

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3074] [Q&A]
ı open lutris and gta game this happened
![2021-12-07-063958_1280x1012_scrot_001](https://user-images.githubusercontent.com/80429360/144963029-853c45d4-5fc4-4ba4-810d-a8a248fc3a4d.png)
![2021-12-07-064212_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/144963033-1cd796e9-17d4-46d3-8d4b-44fe2c85ec6b.png)
[qtile.log](https://github.com/qtile/qtile/files/7665378/qtile.log)

**Comment by elParaguayo** [comment]
That looks odd!

Unfortunately, it's hard to make much from your log as it's not clear when the issue happened.

Could you do the following:
 - Open a terminal and run `qtile cmd-obj -o cmd -f debug`
 - Start lutris and trigger the crash
 - Go back to the terminal and run `qtile cmd-obj -o cmd -f warning`
 - Post the log from the beginning of the DEBUG section.

**Reply by ghost** [reply]
there is no debug or warning log ? 
![2021-12-07-144919_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/145023987-a1c6e920-b80c-4c38-b9eb-fb555294f800.png)

**Reply by elParaguayo** [reply]
The first command sets the log level to DEBUG. This means there will be more content in your qtile.log file. It doesn't get printed out on the terminal.

**Reply by ghost** [reply]
okay

**Reply by elParaguayo** [reply]
Doesn't look like you opened lutris after setting the log to debug. We need the crash to happen so we can see what's in the logs at that point.

**Reply by ghost** [reply]
sorry ı forget ı open lutris and gta game 
enter this command 
qtile cmd-obj -o cmd -f debug
qtile cmd-obj -o cmd -f warning

but this is empty
![2021-12-07-153747_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/145030432-df965cbc-1145-4e70-8f28-e59ae02cfde2.png)

**Reply by elParaguayo** [reply]
Maybe your log got rotated at that point. Have a look at `~/.local/share/qtile/qtile.log.1`

**Reply by ghost** [reply]
No such file or directory

**Reply by elParaguayo** [reply]
I'd be inclined to restart qtile and then try the steps again.

**Reply by ghost** [reply]
[qtile.log](https://github.com/qtile/qtile/files/7668751/qtile.log)

**Reply by elParaguayo** [reply]
Did lutris crash (like your screenshot)? Not seeing any errors in there.

**Reply by ghost** [reply]
no but sometimes qtile shutdown automatically

**Reply by elParaguayo** [reply]
That's very odd. It would be useful to get the debug log from a time when this does crash (or shutdown) as it's hard to work out whether this is a qtile issue otherwise.

**Reply by ghost** [reply]
okay good day

**Reply by elParaguayo** [reply]
Thanks for reporting the issue, though. Please do keep telling us when you find things that don't work.
